### DISCUSSION TITLE
Creating repositories for packages

### DISCUSSION THREAD

**Discussion by mnaser** [OpenIDC/mod_oauth2#24] [General]
Hi there,

The toolset that you provide is great and super helpful, however, I believe there might be a fair majority of users that are using the distro-shipped versions.  This surely helps with adoption however some modules that have not been published yet couldn't show up in a distro for quite a while.

Are you open to the idea of potentially creating a repository that includes all of the packages that you maintain for OIDC/oauth2?

Thanks,
Mohammed

**Comment by zandbelt** [comment]
packages for the most common distro's are available from the releases sections https://github.com/zmartzone/mod_oauth2/releases
